vroom/system-job.vroom

```diff
@@ -35,9 +47,9 @@ To execute system calls asynchronously, use CallAsync.
   :function Callback(result) abort<CR>
   |  let g:callback_stdout = a:result.stdout<CR>
   |endfunction
-  :let g:invocation = AsyncWait(g:syscall.CallAsync('Callback', 0), 2)
+  :let g:invocation = AsyncWait(g:syscall.CallAsync('Callback', 0), 2) (0.1s)
   ! echo hi
-  :echomsg g:callback_stdout
+  :echomsg g:callback_stdout (0.1s)
```

Comment:
Strange that it's needed on these `:echomsg` lines. What's the effect of not having it? Does making the previous time longer and dropping this one work?

Reply:
Yep, good call.  They were evidently just acting to extend the CallAsync delays, since 100ms wasn't enough.